manifests/rhoai/addon/apps/nvidia/gpu-quickstart.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - You have no workbenches running in Red Hat OpenShift AI.
 
         1. Navigate to the Red Hat OpenShift AI dashboard.
-        2. Click **Launch** on the Jupyter card to open the **Start a workbench** page.
+        2. Click **Launch** on the Jupyter card to open the **Start a basic workbench** page.
```

Comment:
It's "Launch application"

<img width="344" alt="Screenshot 2025-05-07 at 9 53 14 AM" src="https://github.com/user-attachments/assets/68a44268-9d2a-4f21-92e3-4807957e0e01" />